components/TextFields/src/MDCTextInputController.h

```diff
@@ -62,9 +62,16 @@
  The color used to denote error state in the underline, the errorText's label, the placeholder and
  the character count label.
 
+ Default is errorColorDefault.
+ */
+@property(nonatomic, null_resettable, strong) UIColor *errorColor;
+
+/** 
+ Default value for above errorColor.
+ 
  Default is red.
  */
-@property(nonatomic, nullable, strong) UIColor *errorColor UI_APPEARANCE_SELECTOR;
+@property(class, nonatomic, null_resettable, strong) UIColor *errorColorDefault;
```

Comment:
Naming is hard, but wouldn't this be defaultErrorColor?  Type usually comes last, and this is a color.
(Here and below)